.test-infra/jenkins/job_LoadTests_Combine_Python.groovy

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import LoadTestsBuilder as loadTestsBuilder
+import PhraseTriggeringPostCommitBuilder
+
+def now = new Date().format("MMddHHmmss", TimeZone.getTimeZone('UTC'))
+
+def loadTestConfigurations = [
+        [
+                title        : 'Combine Python Load test: 2GB 10 byte records',
+                itClass      : 'apache_beam.testing.load_tests.combine_test:CombineTest.testCombineGlobally',
+                runner       : CommonTestProperties.Runner.DATAFLOW,
+                sdk          : CommonTestProperties.SDK.PYTHON,
+                jobProperties: [
+                        job_name            : 'load-tests-python-dataflow-batch-combine-1-' + now,
+                        project             : 'apache-beam-testing',
+                        temp_location       : 'gs://temp-storage-for-perf-tests/smoketests',
+                        publish_to_big_query: true,
+                        metrics_dataset     : 'load_test',
+                        metrics_table       : 'python_dataflow_batch_combine_1',
+                        input_options       : '\'{"num_records": 200000000,' +
+                                '"key_size": 1,' +
+                                '"value_size": 9,' +
+                                '"bundle_size_distribution_type": "const",' +
+                                '"bundle_size_distribution_param": 1,' +
+                                '"force_initial_num_bundles": 1}\'',
+                        max_num_workers     : 5,
```

Comment:
I know that `max_num_workers` and `num_workers` change `autoscaling_algorithm` to NONE, but maybe it would be good to be sure here and specify `autoscaling_algorithm: "NONE"` just in case, WDYT?